spec/controllers/api/v1/packages_controller_spec.rb

```diff
@@ -38,6 +38,18 @@ def test_packages_location_changes(package)
     expect(package.locations.first.building).to eq 'Dispatched'
   end
 
+  describe "Dispatching item" do
+    before { generate_and_set_token(user) }
+    let(:order) { create :order, state: Order::ORDER_NOT_PROCESSED_STATES.sample}
+    let(:orders_package) { create :orders_package, package: package, order: order }
+
+    it "throws error if order is not processed" do
```

Comment:
Can we check response error message here?

Reply:
yeah sure, it can be done. I'll update changes here.